pkg/sys/isatty_unix.go

```diff
@@ -4,12 +4,11 @@ package sys
 
 import (
 	"os"
-	"unsafe"
 )
 
 // IsATTY returns true if the given file is a terminal.
 func IsATTY(file *os.File) bool {
-	var term Termios
-	err := Ioctl(int(file.Fd()), getAttrIOCTL, uintptr(unsafe.Pointer(&term)))
+	//_, err := unix.IoctlGetTermios(int(file.Fd()), getAttrIOCTL)
```

Comment:
Do not comment out code, simply remove it.

Reply:
Agreed. That was leftover from when I was dithering on whether to rewrite this function in terms of Go's `unix.IoctlGetTermios` or `TermiosForFd`.